### ISSUE TITLE
Left-offset overlays?

### ISSUE THREAD

**Issue by blah238** [nyon/fontawesome-actions#20]
Is it possible to put the icon overlay on the left side of the main icon? For example I'd like a version of the database icon with a right-facing arrow on the left side.

Thanks for your great work!

**Comment by nyon** [comment]
Yes it is. You can modify `main.py` to position the operator (e.g. your arrow) on the left side of the icon.

You can add a rule [here](https://github.com/nyon/fontawesome-actions/blob/59ff3ff0bd4a8f6dcb5c1f7b341c5adc35959fb7/main.py#L182) for `tl` and `bl` for top left and bottom left.
